web/views/error_view.ex

```diff
@@ -9,6 +9,14 @@ defmodule Org.ErrorView do
     "Internal server error"
   end
 
+  def render("401.json", _assigns) do
+    %{error: "Please sign in"}
```

Comment:
Yeah, I don't know, I haven't looked at what the actual response looks like, so maybe Phoenix adds some stuff automatically?